client/js/components/login-form.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import TextField from 'material-ui/lib/text-field';
+import Dialog from 'material-ui/lib/dialog';
+import RaisedButton from 'material-ui/lib/raised-button';
+import UserActions from '../actions/user-actions';
+
+export default class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this._onRequestClose = this._onRequestClose.bind(this);
+    this._onSignInRequest = this._onSignInRequest.bind(this);
+  }
+
+  _onRequestClose() {
+    this.refs.dialog.setState({open: true});
+  }
+
+  _onSignInRequest() {
+    var userData = { email: this.refs.emailInput.getValue(),
+                     password: this.refs.passwordInput.getValue() };
+    UserActions.login(userData);
+  }
+
+  render() {
+    return (
+      <Dialog ref="dialog" defaultOpen={true} onRequestClose={this._onRequestClose}>
+        <form className="registration-form">
```

Comment:
Daria mentioned it would be good to bind `_onSignInRequest` also to form `onSubmit`